README.md

```diff
@@ -1,155 +1,152 @@
+**EOSAuth.LoginConnect()** is a wrapper for the Connect process. It takes user credentials and optional data in the custom `EOSLoginConnectSet` container and provides a callback with the `ResultCode`. You can use **EOSAuth.LoginEpicAccountToProductUser()** to simplify the conversion from `EpicAccountId` to `ProductUserId`. If the connection fails, use **EOSAuth.LoginEpicAccountToProductUserWithCreate()** to create the user if not already linked.
 
-With the provided helper functions your complete login flow could look something like this if you want a solid Epic User verification:
+Your complete login flow might look like this:
 
-1. Run the EOSAuth.LoginAuth() with "Persistent Auth" type (in case user already verified from a previous run) and provide a callback to check the result
-2. If Persistent Auth failed, run EOSAuth.LoginAuth() again in the previous callback with "Exchange Code" type (so that they can be automatically logged in from the EGS launcher, though it doesn't always work) and provide a callback to check the result
-3. If Exchange Code failed, run EOSAuth.LoginAuth() again in the previous callback with "Account Portal" type instead (as a last resort so they can use the web portal to log in; This can happen on first run when the user runs the app from the file browser)
-4. If any of the LoginAuth() attempts was successful, run EOSAuth.LoginEpicAccountToProductUserWithCreate() and provide a callback that stores the resulting ProductUserId
-5. Now you have all the user identification necessary to access a vast majority of the SDK functionality
+1. Use `LoginAuth()` with **Persistent Auth** and provide a callback to check the result.
+2. If it fails, retry with **Exchange Code** login.
+3. If that also fails, use **Account Portal** login as a fallback.
+4. On success, call `LoginEpicAccountToProductUserWithCreate()` and store the `ProductUserId`.
 
-Hopefully this example demonstrates how OpenEOS lets you customize your Auth flow while minimizing the amount of complicated manual conversions needed to complete the Authorization process.
+---
 
 ### EOSEnv Layer
 
-EOSEnv provides some utilities for getting values from the command line, which the EGS Launcher utilizes to tell you information about the running environment.
-
-**EOSEnv.GetAllCommandLineArgs()** will help you debug all of the parameters passed into your app.
-
-**EOSEnv.GetCommandLineArgValue()** returns the value of the parameter when given a name. Epic formats their arguments like -NAME=VALUE, so provide the NAME here to get back the value which is trimmed out internally.
+**EOSEnv** provides utilities for retrieving values from the command line used by the EGS Launcher. 
 
-Here are some automated value getters which you might find helpful:
+- **EOSEnv.GetAllCommandLineArgs()** – Helps debug all parameters passed to the app.  
```

Comment:
Listifying it makes sense, but is inconsistent with the paragraph style API list that was in the previous section. I find it more clear if the style matches what I heard earlier so that it recognizable as function calls